src/App.cpp

```diff
@@ -228,7 +228,7 @@ int MyApp::OnExit()
         m_luaLog->Close();
         // Check to see if we have lua messages
         if (m_luaMessages) {
-            XWordErrorMessage(NULL, _T("Errors occurred.  See log file: %s"), (const wxChar *)GetLuaLogFilename().c_str());
+            XWordErrorMessage(NULL, _T("Errors occurred.  See log file: %s"), GetLuaLogFilename().c_str().AsChar());

```

Comment:
According to http://docs.wxwidgets.org/3.1/overview_unicode.html it looks like the current recommendation is `(const char*)s.mb_str()` which avoids the temporary wxCStrData object . . . although from what I read in the wxString docs, s.wx_str() returns either wchar\* or char\* depending on compile options, so that might be the best option